_guides/project-boards/configure.md

```diff
@@ -256,11 +256,13 @@ However, you also have complete control to change how each card is displayed.
 
 To demonstrate this, let's make the cards for closed tasks more obvious.  We don't need to see all those fields once something is completed.
 
-1. Click the gear button (⚙️) at the top of the project board's profile page.
+1. Navigate to the project board's profile page by clicking the search feature and then selecting project boards.
```

Comment:
For consistency, this would be:

~~~
Navigate to the project board's profile page from **Search >> Project Boards**.
~~~